build.sbt

```diff
@@ -15,7 +15,7 @@ import java.io.File
 // ============================================================================
 
 val scalacVersion = "2.13.8"
-val graalVersion = "21.3.0"
+val graalVersion = "22.1.0"
```

Comment:
Yes, definitely. I want to ultimately get to 22.3.0 version, I just wanted to make this step by step, rather than updating straight to the newest version.